src/components/UserMessageSection.jsx

```diff
@@ -1,32 +1,34 @@
 import React from 'react';
 import PathMap from './PathMap';
-import { Routes, Route, HashRouter as Router } from 'react-router-dom';
+import {
+  Routes,
+  Route,
+  BrowserRouter as Router,
+  createBrowserRouter,
+  RouterProvider,
+  createRoutesFromElements,
+} from 'react-router-dom';
 import UserMessage from './UserMessage';
 
 export function UserMessageSection({ setjsonPathMap, jsonPathMap }) {
-  return (
-    <Router>
-      <Route
-        render={({ location }) => (
-          <Routes location={location}>
-            <Route
-              // pathId - URL param for directly accessing particular path.
-              path='/:pathId?'
-              component={({ match }) => (
-                <div className={'fade-in-to-right'}>
-                  <UserMessage
-                    pathId={match.params.pathId}
-                    setjsonPathMap={(json) => setjsonPathMap(json)}
-                    jsonPathMap={jsonPathMap}
-                  />
-                </div>
-              )}
-            />
-
-            <Route path='/map' component={PathMap} />
-          </Routes>
-        )}
-      />
-    </Router>
+  const router = createBrowserRouter(
+    createRoutesFromElements(
+      <>
+        <Route path="/map" element={PathMap} />
+        <Route
+          path="/AdventureSite"
```

Comment:
I might need some clarification here, from `yarn start` I see that the URL that the app launches on is always `http://localhost:3000/AdventureSite` which is why I needed to specify the path to be `/AdventureSite` here so that the app renders the tutorial journey properly on startup. I'm unsure if this will work as-is with how the site is rendered on the Terasology website itself, especially as the link there uses a hash in the URL but the implementation here has been updated to use a `BrowserRouter`